CONTRIBUTING.md

```diff
@@ -0,0 +1,274 @@
+    ```
+
+* Check out the master branch:
+
+    ```shell
+    git checkout master -f
+    ```
+
+* Delete the local branch:
+
+    ```shell
+    git branch -D my-fix-branch
+    ```
+
+* Update your master with the latest upstream version:
+
+    ```shell
+    git pull --ff origin master
+    ```
+
+
+## <a name="rules"></a> Coding Rules
+To ensure consistency throughout the source code, keep these rules in mind as you are working:
+
+* All features or bug fixes **must be tested**.
+* All lists must be alphabetically ordered.
+* Write & format your code as though only one developer wrote the entire codebase.
+
+## <a name="commit"></a> Commit Message Format
+
+*This specification is inspired by and supersedes the [angular commit message format][commit-message-format].*
+
+We have very precise rules over how our Git commit messages must be formatted.
+This format leads to **easier to read commit history**.
+
+Each commit message consists of a **header**, a **body**, and a **footer**.
+
+
+```
+<header>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+The `header` is mandatory and must conform to the [Commit Message Header](#commit-header) format.
+
+The `body` is mandatory for all commits except for those of type "docs" and "deploy".
+When the body is present it must be at least 20 characters long and must conform to the [Commit Message Body](#commit-body) format.
+
+The `footer` is optional. The [Commit Message Footer](#commit-footer) format describes what the footer is used for and the structure it must have.
+
+Any line of the commit message cannot be longer than 100 characters.
+
+
+#### <a name="commit-header"></a>Commit Message Header
+
+```
+<type>(<scope>): <short summary>
+  │       │             │
+  │       │             └─⫸ Summary in present tense. Not capitalized. No period at the end.
+  │       │
+  │       └─⫸ Commit Scope: do-dev-1|faas|gke-staging-01
+  │
+  └─⫸ Commit Type: build|ci|deploy|docs|feat|fix|perf|refactor|test
+```
+
+The `<type>` and `<summary>` fields are mandatory, the `(<scope>)` field is optional.
+
+##### Type
+
+Must be one of the following:
+
+* **build**: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+* **ci**: Changes to our CI configuration files and scripts (example scopes: Circle, BrowserStack, SauceLabs)
+* **deploy**: Anything causing as deployment of a service
+* **docs**: Documentation only changes
+* **feat**: A new feature
+* **fix**: A bug fix
+* **perf**: A code change that improves performance
+* **refactor**: A code change that neither fixes a bug nor adds a feature
+* **test**: Adding missing tests or correcting existing tests
+
+
+##### Scope
+The scope should be the name of one of the root folders.
+
+Note: DO NOT USE JIRA tickets as scopes! Either use the footer or place them
+at the end of the subject
+
+The following is the list of supported scopes:
+
+Please see each individual repository for a list of acceptable scopes!
```

Comment:
**[BREAKING]** When did we change our stance on this? Last I checked the final word was from CR [here](https://ratehub.slack.com/archives/C507X0LLD/p1617853147112800?thread_ts=1615840771.098800&cid=C507X0LLD)

 ```
Final word on commit convention and (scope) in particular is... do what you want.

The JIRA issue must be noted in the commit message somewhere. 
I prefer to use scope for 'area of codebase' but unless it ends up causing process issues I don't feel strongly enough to mandate something here. 
Also note that scope is optional so you can leave out the (...) but not the : 
```

Imo this should just read `Please see each individual repository for a list of acceptable scopes!` to be safe :+1:

Reply:
Already changed, was a leftover from migrating this doc from ratehub-k8s 🗡️ 

Definitely per repo for scopes